src/py/flwr/server/app.py

```diff
@@ -345,7 +343,7 @@ def run_superlink() -> None:
         # Scheduler thread
         scheduler_th = threading.Thread(
             target=_flwr_serverapp_scheduler,
-            args=(state_factory, args.driver_api_address),
+            args=(state_factory, args.driver_api_address, args.ssl_ca_certfile),
```

Comment:
I believe `ssl_ca_certfile` is for node authentication. I am not super sure here but why we can use this as the root certificates for the ServerApp process? Shouldn't we use `root_certificates`?

Reply:
there is no `root_certificates` in the `SuperLink`...

Reply:
@jafermarq Just checked! Yes, we can let the subprocess use ssl_ca_certfiles. I'll resolve this.